oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneDocumentMaker.java

```diff
@@ -88,7 +88,7 @@ public LuceneDocumentMaker(@Nullable FulltextBinaryTextExtractor textExtractor,
     @Override
     protected void indexAnalyzedProperty(Document doc, String pname, String value, PropertyDefinition pd) {
         String analyzedPropName = constructAnalyzedPropertyName(pname);
-        doc.add(newPropertyField(analyzedPropName, value, !pd.skipTokenization(pname), pd.stored));
+        doc.add(newPropertyField(analyzedPropName, value, !pd.skipTokenization(pname), pd.useInExcerpt));
```

Comment:
not sure about this change

Reply:
I've reverted even thought I still don't see the why stored is even ok here :)

Reply:
`oak-search`  could be potentially used by an external project as a dependency. If you want to change a public member, you have to a) deprecate it b) remove it in a successive major release. My suggestion is to use `stored`.